assets/details-disclosure.js

```diff
@@ -39,6 +39,8 @@ class HeaderMenu extends DetailsDisclosure {
   }
 
   onToggle() {
+    document.body.classList.toggle('overflow-hidden', this.querySelectorAll('.mega-menu[open]').length);
```

Comment:
this class, when applied and not only in this scenario, is creating a bit of a jump that makes the experience feel subpar 🤔 

It does it as well on the drawer for pick up availability and probably anytime we apply the `overflow-hidden` class in the theme. 

[video](https://screenshot.click/29-57-wcssz-97g0x.mp4)

Reply:
When applying the `overflow: hidden;` it hides the scroll bar from the page so it creates that shift.

Reply:
One option that pops into my head but which I'm not sure would work... `overflow-y: scroll` might force a scrollbar (again, might depend on OS), and then make the height of the container `100vh` so that it has nowhere to go... you'd also need to add a `transform` to it to position it in the spot it was in before adding the overflow.  Feels complicated, but something like that might work. 🤷 

We can look into if there are any more elegant solutions.

Reply:
Hmm yeah I was reading suggestions yesterday but nothing seemed very good or straight forward 🤔 
This sounded interesting but didn't try it yet: https://aykevl.nl/2014/09/fix-jumping-scrollbar